Gems/SceneLoggingExample/Code/SceneLoggingExampleModule.cpp

```diff
@@ -42,7 +42,7 @@ namespace SceneLoggingExample
         }
 
         // In this example, no system components are added. You can use system components 
-        // to set global settings for this gem from the Project Configurator.
+        // to set global settings for this gem from the Project Manager.
```

Comment:
Unless this means enabling or disabling a gem Project Manager can't do that

Reply:
Will remove this mention outright as well